relay-server/src/services/processor/event.rs

```diff
@@ -145,10 +149,37 @@ pub fn extract<Group: EventProcessing>(
 
     metrics.bytes_ingested_event = Annotated::new(event_len as u64);
 
+    if let Some(event_value) = event.value() {
```

Comment:
This is the wong place to do this:
- we should do like everything else, have all the logs stuff in a dedicated module
- this exact functions' purpose is to generate an event from it's various incarnations, nothing else